cli/index.ts

```diff
@@ -132,12 +133,20 @@ const promptOptions = async (
     });
```

Comment:
We should remove Python as an option if GraphQL was chosen as the API type through the command line args, kinda weird that we still present the option but error out at the very end